interface-definitions/service_pppoe-server.xml.in

```diff
@@ -70,19 +70,25 @@
             <children>
               <leafNode name="vlan-id">
                 <properties>
-                  <help>VLAN monitor for the automatic creation of vlans (user per vlan)</help>
-                  <constraint>
-                    <validator name="numeric" argument="--range 1-4096"/>
+                  <help>VLAN monitor for the automatic creation of single vlan</help>
+                  <valueHelp>
+                    <format>u32:1-4094</format>
+                    <description>VLAN monitor for the automatic creation of single vlan</description>
+                  </valueHelp>                  <constraint>
+                    <validator name="numeric" argument="--range 1-4094"/>
                   </constraint>
-                  <constraintErrorMessage>VLAN ID needs to be between 1 and 4096</constraintErrorMessage>
+                  <constraintErrorMessage>VLAN ID needs to be between 1 and 4094</constraintErrorMessage>
                   <multi/>
                 </properties>
               </leafNode>
               <leafNode name="vlan-range">
                 <properties>
-                  <help>VLAN monitor for the automatic creation of vlans (user per vlan)</help>
-                  <constraint>
-                    <regex>(409[0-6]|40[0-8][0-9]|[1-3][0-9]{3}|[1-9][0-9]{0,2})-(409[0-6]|40[0-8][0-9]|[1-3][0-9]{3}|[1-9][0-9]{0,2})</regex>
+                  <help>VLAN monitor for the automatic creation of vlans range</help>
+                  <valueHelp>
+                    <format>start-end</format>
+                    <description>VLAN monitor range for the automatic creation of vlans (e.g. 1-4094)</description>
+                  </valueHelp>                  <constraint>
```

Comment:
oops - `<constraint>` moved into the wrong line.

Reply:
Done.